research/delf/delf/__init__.py

```diff
@@ -35,5 +35,4 @@
 from delf.python import detect_to_retrieve
 from delf.python import training
 from delf.python.training import model
-from delf.python.training import datasets
```

Comment:
just wanted to double-check: after doing this change, the code continues to work, right? I thought some import like this in the top init could still be needed.

Reply:
The code was running, but I now added a new updated import just to make sure it works the same way